microservices-schema.json

```diff
@@ -0,0 +1,120 @@
+                "examples": [
+                    "Payment methods provided to the customer"
+                ]
+            },
+            "repository": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/repository",
+                "title": "API repository",
+                "description": "Link to the API repository",
+                "default": null,
+                "examples": [
+                    "https://github.com/hmcts/payments"
+                ]
+            },
+            "spec": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/spec",
+                "title": "API specs",
+                "description": "Link to API specification json file",
+                "default": null,
+                "examples": [
+                    "https://hmcts.github.io/reform-api-docs/specs/payments.json"
+                ]
+            },
+            "colour": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/colour",
+                "title": "Network colout",
+                "description": "Colour used to paint and group APIs in the network graph",
+                "default": "",
+                "examples": [
+                    "#8e44ad"
+                ]
+            },
+            "dependencies": {
+                "type": "array",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies",
+                "items": {
+                    "type": "object",
+                    "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items",
+                    "properties": {
+                        "name": {
+                            "type": "string",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/name",
+                            "title": "API name",
+                            "description": "The name of the API, usually a match of GitHub repo README title",
+                            "default": "",
+                            "examples": [
+                                "IdAM"
+                            ]
+                        },
+                        "hard": {
+                            "type": "boolean",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/hard",
+                            "title": "???",
+                            "description": "???",
```

Comment:
provide title and description

Reply:
I feel like schema is a bit too big. I don't know what certain fields suppose to mean - do they have any value at all, but basically I took the existing data blob and generated schema for it. Therefore certain values I do not know how to fill in and leave that to @jasonpaige.